src/constants.ts

```diff
@@ -138,6 +138,19 @@ export const COOKIE_SECRET = getEnvironmentValue("COOKIE_SECRET");
 // Booleans
 export const TRUE = "true";
 
+// Matomo
+export const MATOMO_LINK_CLICK = "Link Click";
+export const MATOMO_BUTTON_CLICK = "Button Click";
+export const MATOMO_ADD_COMPANY_BUTTON = "Add a Company Button";
+export const MATOMO_VIEW_AND_MANAGE_LINK = "View and Manage Link";
+export const MATOMO_CONTINUE_BUTTON = "Continue Button";
+export const MATOMO_CONFIRM_AND_CONTINUE_BUTTON = "Confirm and Continue Button";
+export const MATOMO_CHOOSE_DIFFERENT_COMPANY_LINK = "Choose a Different Company Link";
+export const MATOMO_ADD_NEW_AUTHORISED_PERSON_BUTTON = "Add a New Authorised Person Button";
+export const REMOVE_AUTHORISED_USER = "Remove Authorised User";
```

Comment:
Add MATOMO_ to constants to make it clear what this is all about and prevent mix ups with other stuff. Also add the work Link to the tag text.